docs/knowledgebase/getting-started/index.md

```diff
@@ -84,12 +82,70 @@ curl https://sh.rustup.rs -sSf | sh
 source ~/.cargo/env
 ```
 
-Configure the Rust toolchain to default to the latest stable version:
+Configure the Rust toolchain to default to the latest stable version, add nightly and the nightly wasm target:
 
 ```bash
 rustup default stable
+rustup update
+rustup update nightly
+rustup target add wasm32-unknown-unknown --toolchain nightly
 ```
 
+## Test Your Set-Up!
+
+Now the best way to ensure that you have successfully prepared a computer for Substrate
+development is to follow the steps in our first tutorial:
+
+**\> [Create Your First Substrate Chain](../../tutorials/create-your-first-substrate-chain/) <**
+
+--- 
+
+> **For more details on _why_ these dependancies are used, and for *troubleshooting* errors building the template, read on.**
+
+---
+
+## Troubleshooting Substrate Builds
+
+Sometimes you con't get the [substrate developer hub](https://github.com/substrate-developer-hub/) templates to compile out of
+the box. Here is some tips to help you work through that.
+
+### Rust Configuration Check
+
+To see what Rust toolchain you are presently using, run:
+
+```bash
+rustup show
+```
+
+```bash
```

Comment:
I don't feel too strongly about this but the idea is to be more explicit. 
```suggestion
```

This should echo:
```bash
```

Reply:
The intention is to let users use the "copy" icon button to get just what they need, but also to have to snippet of the output stand out as a code block. What would you do here for that?

Reply:
Rectified this in the above comment. Ignore this. =)